addons/auth_signup/data/mail_template_data.xml

```diff
@@ -249,5 +249,103 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Email template for new portal users -->
+        <record id="portal_set_password_email" model="mail.template">
+            <field name="name">Settings: New Portal User Invite</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">Your account at {{ object.company_id.name }}</field>
+            <field name="email_to">{{ object.email_formatted }}</field>
+            <field name="description">Sent to new portal user after you invited them</field>
+            <field name="body_html" type="html">
+                <table border="0" cellpadding="0" cellspacing="0"
```

Comment:
You can start left, so that it is easier to read (no need to keep indent level after field for html content)